build/rsync/build.sh

```diff
@@ -18,12 +18,12 @@
 . ../../lib/build.sh
 
 PROG=rsync
-VER=3.4.0
+VER=3.4.1
 PKG=network/rsync
 SUMMARY="rsync - faster, flexible replacement for rcp"
 DESC="An open source utility that provides fast incremental file transfer"
 
-XXHASHVER=0.8.1
+XXHASHVER=0.8.3
```

Comment:
Just for my own understanding, why did this change?

Reply:
Just a maintenance bump of the xxhash dependency to the latest version while I was in here. There wasn't a specific bug fix requirement, although there are a number of fixes after the 0.8.1 we had.

Reply:
Gotcha, thanks for the clarification.